lisa/features/gpu.py

```diff
@@ -215,14 +245,13 @@ def get_gpu_count_with_lspci(self) -> int:
     def get_gpu_count_with_vendor_cmd(self) -> int:
         device_count = 0
 
-        if "nvidia" in self.gpu_vendor:
-            result = self._node.execute("nvidia-smi")
-            if result.exit_code != 0 or (result.exit_code == 0 and result.stdout == ""):
-                raise LisaException(
-                    f"nvidia-smi command exited with exit_code {result.exit_code}"
-                )
-            for device_info in result.stdout.splitlines():
-                if any(device_info in device_name for device_name in self.gpu_devices):
-                    device_count += 1
+        result = self._node.execute("nvidia-smi")
```

Comment:
Sorry, haven't catch that. If the GPU is from AMD, there is no nvidia-smi. So the original logic check if nvidia is in vendor or not. What's the reason to remove it, and is it tested with AMD gpu?

Reply:
let me double check, origin design doesn't work, since it doesn't install the driver, and self.gpu_vendor is empty, let me try to add it back.